sympy/core/function.py

```diff
@@ -1769,6 +1769,40 @@ def _eval_as_leading_term(self, x):
         # The variable is independent of the substitution:
         return self.expr.as_leading_term(x)
 
+    def fsubs(self, both):
+        """this function is used to Solve the floor and ceiling derivatives
+         both = True if the expression contain both floor and ceiling
+         both = False if the expression contain only floor or ceiling expressions"""
+
+        from sympy.printing import StrPrinter
+        from sympy import floor
+        from sympy import ceiling
+        from sympy import sympify
+        total = 0
+        m = str(self).replace(" + ", "!")
```

Comment:
I suspect this can be solved without strings. What (in words) are you trying to do?